src/tests/e2e.test.ts

```diff
@@ -0,0 +1,291 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { getTokenByChainAndSymbol } from '@indexcoop/tokenlists'
+import { expect } from 'chai'
+import { ETH } from 'constants/tokens'
+import { ethers } from 'ethers'
+
+import inputTokenMap from './inputTokenMap.json'
+import rpcConfig from './rpcConfig.json'
+import scenarios from './testScenarios.json'
+
+import {
+  FlashMintContractType,
+  FlashMintQuoteProvider,
+  type FlashMintQuoteRequest,
+} from '../quote/provider'
+
+import {
+  getLifiSwapQuoteProvider,
+  getZeroExV2SwapQuoteProvider,
+  wei,
+} from './utils'
+
+// upstream Alchemy key for forking
+const ALCHEMY_KEY = process.env.ALCHEMY_API_KEY
+if (!ALCHEMY_KEY) throw new Error('Please set ALCHEMY_API_KEY')
+
+// drive all JSON-RPC through this local node
+const LOCAL_RPC_URL = process.env.LOCAL_RPC_URL || 'http://127.0.0.1:8545'
+const localProvider = new JsonRpcProvider(LOCAL_RPC_URL)
+
+// safe fork blocks
+const blockNumbers: Record<string, number> = {
+  '1': 22322000,
+  '8453': 29255000,
+  '42161': 328930000,
+}
```

Comment:
From my experience with the SDK it was always good to run against `latest` to potentially see some new issues. Although this way the tests are more stable, I guess. We probably should watch what to favor here - as development of the SDK progresses.